File: test/single-language/csharp/dotnetsdk/http-service/success-runtime-config/Models/Summary.cs
using Newtonsoft.Json;
namespace Fixture;

public sealed class Summary
{
    [JsonProperty("status")] public string Status => "ok";
    [JsonProperty("items")] public IReadOnlyList<int> Items { get; }
    [JsonProperty("total")] public int Total => Items.Sum();
    public Summary(IEnumerable<int> items) { Items = Array.AsReadOnly(items.ToArray()); }
    public string ToJson() => JsonConvert.SerializeObject(this);
}

File: test/single-language/go/gomodule/http-service/success-runtime-config/internal/model/summary.go
package model

type Summary struct {
	Status string `json:"status"`
	Items  []int  `json:"items"`
	Total  int    `json:"total"`
}

func NewSummary(items []int) Summary {
	result := Summary{Status: "ok", Items: append([]int(nil), items...)}
	for _, item := range result.Items {
		result.Total += item
	}
	return result
}
